modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/ResourceManager.java

```diff
@@ -352,4 +352,101 @@ default SCIMResponse updateWithPATCHRole(String id, String patchRequest, RoleV2M
         return new SCIMResponse(ResponseCodeConstants.CODE_NOT_IMPLEMENTED, ResponseCodeConstants.DESC_NOT_IMPLEMENTED,
                 Collections.emptyMap());
     }
+
+    /**
+     * POST method to create a new role in roleV2 model.
```

Comment:
Let's specify create role without user or group assignments. The method comment will imply the difference between the previous method and new method.

Reply:
Fixed with https://github.com/wso2/charon/pull/435/commits/b30c47b5a19cfbad584e1b89e99b83795cb2a7c5